### PR TITLE
Fix imports handling

### LINKED ISSUE #773: Missing Box_ import in some cases
In a progress callback of gio, we have a missing import of `Box_`. Will put back the name of the function when I remember it. :-.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #773.

cc @sdroege @EPashkin

### CODE REVIEW

File: src/analysis/imports.rs [hunk 1]
```diff
@@ -74,21 +77,31 @@ impl Imports {
...
                 }
+                entry
+            } else {
+                let entry = self
+                    .map
+                    .entry(name.to_owned())
+                    .or_insert((version, Vec::new()));
+                // Since there is no constraint on this import, if any constraint
+                // is needed, we can just remove it.
+                entry.1 = Vec::new();
```
Comment (EPashkin) [inline src/analysis/imports.rs]: Or call `clear`

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks, import part looks good.

Part related to https://github.com/gtk-rs/gtk/issues/615 is unfinished?

File: src/analysis/imports.rs (hunk 1 above)
Comment (EPashkin) [reply src/analysis/imports.rs]: or set `Vec::new` instead `clear` in `add` too.

**Comment by GuillaumeGomez** [comment]
gtk-rs/gtk#615 is "finished": we need to regenerate docs and release a new version of `lgpl-docs`.

**Comment by EPashkin** [comment]
@GuillaumeGomez I meant that this PR contains only debug prints in src/codegen/doc/mod.rs

**Comment by GuillaumeGomez** [comment]
Oh crap! Completely forgot to remove them... Done! And yes, I was trying to figure out why we had this issue. It's not an issue anymore, just need a rebuild.

**Pull request merged**